modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -3920,7 +3920,7 @@
 		"newImports": [
 			"com.liferay.document.library.kernel.model.DLVersionNumberIncrease"
 		],
-		"to": "updateFileEntry(param#0#, param#1#, param#2#, param#3#, param#4#, null, param#5#, param#6#, dlVersionNumberIncrease, param#7#, null, null, param#8#, param#9#)"
+		"to": "updateFileEntry(param#0#, param#1#, param#2#, param#3#, param#4#, null, param#5#, param#6#, dlVersionNumberIncrease, param#8#, null, null, null, param#9#)"
```

Comment:
@regiCesar21 the `DLVersionNumberIncrease` isn't making any sense here, where is being created?